src/app/launcher/step-indicator/step-indicator.component.ts

```diff
@@ -70,6 +73,14 @@ export class StepIndicatorComponent implements OnInit {
     }, 10);
   }
 
+  /**
+   * Validate the application name
+   */
+  validateProjectName(): void {
+    this.launcherComponent.isProjectNameValid =
+      this.dependencyCheckService.validateProjectName(this.launcherComponent.summary.dependencyCheck.projectName);
+  }
+
```

Comment:
This can wait for another PR, but validateProjectName could be added to the launcher component so the validation call is in one place.

Considering that we're already referencing launcherComponent in step-indicator.component.html, for example:

[(ngModel)]="launcherComponent.summary.dependencyCheck.projectName"

We could just add this:

(keyup)="launcherComponent.validateProjectName()"

Reply:
Yeah. I agree. Nice suggestion.

Reply:
@jyasveer I feel it'll be good to have an issue for this as a reminder